src/app/theme.ts

```diff
@@ -9,6 +9,18 @@ export interface ColorSet {
   darkest?: string;
 }
 
+export const hiddenButAccessible = `
```

Comment:
Nitpicking but `visuallyHidden` could be a good name for this

Reply:
There's a `visuallyHidden` [in Dropdown](https://github.com/openstax/rex-web/blob/make-group-for-dropdown-checkboxes/src/app/components/Dropdown.tsx#L49) (and a contrasting `visuallyShown`). Since it uses height/width `0`, it's also hidden from screenreaders. Probably need to rename it to `fullyHidden` or something. I'm not entirely clear on what the point of it is, why it's not `display: none`.